minvws/nl-kat-coordination
# Add audit trail logging to boefje crud actions in boefje

### Changes

This PR adds logging when a boefje (variant) is being created or edited.

### Issue link
Closes #3603 

### Demo
Creating Boefje:
![afbeelding](https://github.com/user-attachments/assets/a9c5443b-8ce1-487e-a9e3-a41c6a48f8a0)

Editing Boefje:
![afbeelding](https://github.com/user-attachments/assets/59584f3c-e09a-41e6-b7ef-b4e7e8f7c56c)

### QA notes
- Create a new Boefje, check the rocky logs
- Create a new Boefje variant, check the rocky logs
- Edit an existing Boefje, check the rocky logs

---

### Code Checklist







---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.
